app/views/admins/_user_entry.haml

```diff
@@ -1,70 +1,85 @@
-
 %li.user.media
-  %div.pull-left
-    - if user.person
-      %span.media-object
-        = person_image_tag(user.person, size: :thumb_small)
+  .media-body.row
+    .col-md-1
+      - if user.person
+        %span.media-object
+          = person_image_tag(user.person, size: :thumb_small)
```

Comment:
I tried to fix that with https://github.com/Flaburgan/diaspora/commit/b7bde63be75a0d5e653fa857864d35b35ae48615#diff-9d44402b8caede0852040adab1428922R5 but the avatar is not displayed anymore even if it has the 50x50 size fixed by CSS as before. Any idea what's going on?